CMakeLists.txt

```diff
@@ -110,25 +110,28 @@ if( EXIV2_BUILD_SAMPLES )
             WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
             COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bash_tests
         )
-        add_test(NAME bugfixTests
-            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bugfixes
-        )
-        add_test(NAME lensTests
-            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose lens_tests
-        )
-        add_test(NAME tiffTests
-            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose tiff_test
-        )
-        add_test(NAME versionTests
-            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bash_tests/version_test.py 
-        )
     endif()
 endif()
 
+if (Python3_Interpreter_FOUND)
+    add_test(NAME bugfixTests
+        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
+        COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bugfixes
+    )
+    add_test(NAME lensTests
+        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
+        COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose lens_tests
+    )
+    add_test(NAME tiffTests
+        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
+        COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose tiff_test
+    )
+    add_test(NAME versionTests
+        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
+        COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bash_tests/version_test.py 
+    )
+endif()
```

Comment:
Let's add a warning message in an `else` block here just so we don't silently not add any tests.